### PR TITLE
Update to zoom in to where user double tapped

### PR DESCRIPTION
**Pull request by nuudles**
This pull request updates the component to zoom in on the location that the user double tapped rather than just focusing on the center.

### CODE REVIEW

**Review by muukii — COMMENTED** [review]
Thank you.
It seems this feature is good.

File: ZoomImageView/ZoomImageView.swift [hunk 1]
```diff
@@ -31,7 +31,7 @@ open class ZoomImageView : UIScrollView, UIScrollViewDelegate {
 
   // MARK: - Properties
 
-  private let imageView = UIImageView()
+  public let imageView = UIImageView()
```
Comment (muukii) [inline ZoomImageView/ZoomImageView.swift]: @nuudles 
Does it need to change to public?

Comment (nuudles) [reply ZoomImageView/ZoomImageView.swift]: No, I used that for my own purposes to expose the `imageView` to animate for a transition. Feel free to leave it as `private`.

Comment (muukii) [reply ZoomImageView/ZoomImageView.swift]: Okay.
then, could you change back this to private?

**Comment by muukii** [comment]
okay thanks!

**Pull request merged**